net-misc/sunshine/sunshine-0.23.1.ebuild

```diff
@@ -137,7 +137,7 @@ CDEPEND="
 		sys-libs/libcap
 		x11-libs/libdrm
 	)
-	svt-av1? ( media-libs/svt-av1:= )
+	svt-av1? ( <media-libs/svt-av1-3:= )
```

Comment:
This is fine as-is but, on a side-note, don't know if you did this while being aware but I'll note it anyway.

`pkgcheck scan --commits` will give:
```
net-misc/sunshine
  RdependChange: version 0.23.1: RDEPEND modified without revbump
```
..but that is actually fine to ignore (aka not revbump) in this particular case because portage will be forced to look at the updated dependency through  the `:=` with subslot change rather than rely on VDB.

Reply:
Yeah, I knew this. I personally try to keep to the thought process where I need to justify why a change doesn't require a revbump.